pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Mmmh it's strange to downgrade a lockfile

Reply:
yes it is ... I put it back like before